### ISSUE TITLE
Can't edit ALT text when editing a post

### ISSUE THREAD

**Issue by davglass** [mastodon/mastodon#20887]
### Steps to reproduce the problem

1. Edit a Post with an image
2. You can delete the image but not edit the ALT text
3.
...

Editing a post with an image:
<img width="288" alt="Screenshot 2022-11-16 at 9 27 32 AM" src="https://user-images.githubusercontent.com/32551/202222675-791b0404-ca66-4b50-bf1b-217f2d4f9532.png">

Creating a new post with an image:
<img width="288" alt="Screenshot 2022-11-16 at 9 28 19 AM" src="https://user-images.githubusercontent.com/32551/202222671-84e7f8d3-efdb-4cec-bd5f-80be58a8b6e3.png">

### Expected behaviour

You should be able to edit the ALT text on an existing image in a post

### Actual behaviour

There doesn't seem to be a way to edit the ALT of an already posted image.

### Detailed description

Posted screenshots of editing a post with an image, you can't edit the ALT text of an existing image.

### Specifications

Mastodon 4.0.2

**Comment by ineffyble** [comment]
This will be fixed by #20878 which is in-progress.

**Comment by sevenbitbyte** [comment]
Just linking to an additional user posting.

Thanks for the update @ineffyble 

https://legal.social/@krisnelson/109395653335266609

**Comment by Cassolotl** [comment]
This was fixed in *Add support for editing media description and focus point of already-posted statuses #20878* in January, so this issue can now be closed. :)

**Comment by mcclure** [comment]
Issue still open fyi, despite fix @renchap
